inc/saiport.h

```diff
@@ -2164,6 +2164,28 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_MAX_FEC_SYMBOL_ERRORS_DETECTABLE,
 
+    /**
+     * @brief Gather port debug information.
+     *
+     * Gather vendor-specific debug information about the port. The returned
+     * data should be in the form of a JSON-encoded string.
+     *
+     * Example possible response:
+     * {
+     * "port_index": "123",
+     * "lane_1": {
+     * "serdes_fw_rev": "B345",
+     * "dc_offset": "1"
+     * "eye_opening": "125, 183, 46, 46"
+     * },
+     * "additional_debug_data": "Some\n additional\n information"
+     * }
+     *
+     * @type sai_s8_list_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_DEBUG_DATA,
```

Comment:
Perhaps SAI_PORT_ATTR_JSON_DEBUG_DATA?

Reply:
Clarify in comments.
Standard attribute method to get vendor specific data in JSON format.

Change the string to 
SAI_PORT_ATTR_JSON_FORMATTED_DEBUG_DATA

Reply:
Updated, PTAL to make sure the revised wording works.